packages/internal/src/paths.ts

```diff
@@ -74,7 +74,9 @@ const PATH_WEB_DIR_CONFIG_POSTCSS = 'web/config/postcss.config.js'
 /**
  * Search the parent directories for the Redwood configuration file.
  */
-export const getConfigPath = (cwd: string = process.cwd()): string => {
+export const getConfigPath = (
+  cwd: string = process.env.__REDWOOD__CONFIG_PATH ?? process.cwd()
+): string => {
```

Comment:
Use this env-var to point to the fixtures directory.

Reply:
I didn’t quite understand your instructions here (or use case). I got things working and used Copy and Watch tools 👍